README.md

```diff
@@ -34,16 +34,21 @@ cookiecutter projects.
     * Runs cookiecutter to create the project from the template.
 	* Creates a repository on GitHub for the project.
 	* Pushes the project content to GitHub
-  It returns a JSON structure with one field "github_repo", which
-  contains the HTTPS clone url of the new repository.
+
+It returns a JSON structure with two fields: `github_repo`
+  contains the HTTPS clone url of the new repository, and 
+  `post_commit_error` contains either `null` or a string describing any
+  errors that occurred after the project was pushed to GitHub.
+
```

Comment:
That's the only error that is permitted after the repository is successfully pushed to GitHub.  Sadly, Travis CI is sufficiently flaky that it does indeed sometimes happen.